tools/c7n_azure/c7n_azure/utils.py

```diff
@@ -104,31 +104,36 @@ class ThreadHelper:
     disable_multi_threading = False
 
     @staticmethod
-    def execute_in_parallel(resources, execution_method, executor_factory, log,
+    def execute_in_parallel(resources, event, execution_method, executor_factory, log,
```

Comment:
Is event a required parameter?

Reply:
I suppose it gets passed through from process() in AzureActionBase which kicks all this off. Would we ever use this helper method other than in the AzureActionBase class where we might want to make the event optional here?

Reply:
I wouldn't expect any other users for this method.. All Actions supposed to inherit from BaseAction and I don't know if parallel makes sense for filters. I assume, filters also can use this event.